alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -169,6 +184,16 @@ abstract class BaseAlertPresenter(private val alert: Alert) : AlertActions {
             addAction(Alert.Action(title, Alert.Action.Style.NEUTRAL, handler))
         }
 
+        /**
+         * Initializes alert's input field
+         *
+         * @param hint The input field hint
+         * @param textWatcher The callback for text change events of inout field
+         */
+        fun addTextInput(hint: String?, textWatcher: AlertTextWatcher) = apply {
```

Comment:
If you call it twice it will override previous action, it's not obvious

Reply:
Changed to set*